ddclient.in

```diff
@@ -490,17 +490,32 @@ my %variables = (
         'host'                => setv(T_STRING,1, 1, '',                   undef),
 
         'use'                 => setv(T_USE,   0, 0, 'ip',                 undef),
+        'usev4'               => setv(T_USEV4, 0, 0, 'disabled',           undef),
+        'usev6'               => setv(T_USEV6, 0, 0, 'disabled',           undef),
         'if'                  => setv(T_IF,    0, 0, 'ppp0',               undef),
+        'ifv4'                => setv(T_IF,    0, 0, 'default',            undef),
+        'ifv6'                => setv(T_IF,    0, 0, 'default',            undef),
         'web'                 => setv(T_STRING,0, 0, 'dyndns',             undef),
         'web-skip'            => setv(T_STRING,0, 0, '',                   undef),
         'web-ssl-validate'    => setv(T_BOOL,  0, 0, 1,                    undef),
+        'webv4'               => setv(T_STRING,0, 0, 'googledomains',      undef),
+        'webv4-skip'          => setv(T_STRING,1, 0, '',                   undef),
+        'webv6'               => setv(T_STRING,0, 0, 'googledomains',      undef),
```

Comment:
I don't think we should fallback to googledomains here

Reply:
I used googledomains as default value because it's the default value on global option.

dyndns, the default value for `web` option is not available on ipv6

We can use another provider, but for consistency, it shoulld be a good idea to use the same on both global/local option ?
Maybe nsupdate.info as it's a free service https://github.com/nsupdate-info/nsupdate.info available on ipv4 and ipv6, available on https and hosted on a country that respect user data (Germany).

What did you prefer ?

Reply:
Oh, yeah, didn't remember that.